js/src/forum/fragments/PostQuoteButton.js

```diff
@@ -0,0 +1,55 @@
+import Fragment from 'flarum/Fragment';
+import icon from 'flarum/helpers/icon';
+
+import reply from '../utils/reply';
+
+export default class PostQuoteButton extends Fragment {
+  constructor(post) {
+    super();
+
+    this.post = post;
+  }
+
+  view() {
+    return (
+      <button class="Button PostQuoteButton" onclick={() => {
+        this.hide();
+        reply(this.post, this.content);
+      }} mousedown={(e) => e.stopPropagation()}>
+        {icon('fas fa-quote-left', { className: 'Button-icon' })}
+        {app.translator.trans('flarum-mentions.forum.post.quote_button')}
+      </button>
+    );
+  }
+
+  oncreate(vnode) {
+    super.oncreate(vnode);
+
+    $(document).on('mousedown', this.hide.bind(this));
```

Comment:
We should unbind this somehow when destroying the component.

That seems to be a general problem with "fragments"... :thinking:

Reply:
We can still use onremove